hostscripts/gatehost/freshadminvm

```diff
@@ -20,7 +20,7 @@ rm -f mkcloud.config
 : ${cachedir:=/root/tmp}
 mkdir -p $cachedir
 export mkclouddriver=libvirt
-SCRIPTS_DIR=/usr/src/automation/scripts
+test -n "$SCRIPTS_DIR" || SCRIPTS_DIR=/usr/src/automation/scripts
```

Comment:
you could use `: ${SCRIPTS_DIR:=/usr/src/automation/scripts}` notation, same as cachedir above

Reply:
done